clio-server/src/main/resources/logback.xml

```diff
@@ -4,7 +4,7 @@
 
     <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
         <encoder>
-            <pattern>[%level] %msg%n</pattern>
+            <pattern>%-5marker [%level] %msg%n</pattern>
```

Comment:
Seems odd to me that we don't include a timestamp in the log, is that prepended somewhere else?